client/post-editor/media-modal/detail/detail-item.jsx

```diff
@@ -23,21 +24,17 @@ import { userCan } from 'lib/site/utils';
 import versionCompare from 'lib/version-compare';
 import MediaUtils, { isItemBeingUploaded } from 'lib/media/utils';
 import config from 'config';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteOption, isJetpackModuleActive, isJetpackSite } from 'state/sites/selectors';
 
 /**
  * This function return true if the image editor can be
  * enabled/shown
  *
- * @param  {object} item - media item
  * @param  {object} site - current site
- * @return {boolean} `true` is the image-editor can be enabled.
+ * @return {boolean} `true` if the image-editor can be enabled.
  */
-const enableImageEditing = ( item, site ) => {
-	// do not allow if, for some reason, there isn't a valid item yet
-	if ( ! item ) {
-		return false;
-	}
-
```

Comment:
Hmmm...no, that looks wrong. I'm guessing this was some sort of rebasing error. Put it back in https://github.com/Automattic/wp-calypso/pull/11653/commits/cc6f74236aea6d2fb4aca40cd003135c1595fa57.